lib/experimental/Utilities/PrivateBox/index.stories.tsx

```diff
@@ -0,0 +1,43 @@
+// Replace your-framework with the name of your framework
+import { Button } from "@/../lib/components/Actions/Button"
+import { PrivacyModeProvider, usePrivacyMode } from "@/lib/privacyMode"
+import type { Meta, StoryObj } from "@storybook/react"
+import { PrivateBox } from "."
+
+const meta: Meta = {
+  decorators: [
+    (Story) => (
+      <PrivacyModeProvider>
```

Comment:
We are already adding the `FactorialOneProvider` in all stories, so this is redundant now :)